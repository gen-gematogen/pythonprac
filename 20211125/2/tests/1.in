ÂÑÅÌ ÏÐÈÂÅÒ ÄÎÐÎÃÈÅ ÄÐÓÇÜß Ñ ÂÀÌÈ ÄÅÌÎÍ È ÀÍÄÐÎÉÄ