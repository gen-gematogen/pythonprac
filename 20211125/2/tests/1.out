ВСЕМ ПРИВЕТ ДОРОГИЕ ДРУЗЬЯ С ВАМИ ДЕМОН И АНДРОЙД
